precise/iaf_psc_alpha_canon.cpp

```diff
@@ -169,8 +169,6 @@ nest::iaf_psc_alpha_canon::State_::get( DictionaryDatum& d, const Parameters_& p
   def< double >( d, names::V_m, y3_ + p.E_L_ ); // Membrane potential
   def< double >( d, "y1", y1_ );                // y1 state
   def< double >( d, "y2", y2_ );                // y2 state
```

Comment:
I agree that those lines should be removed or get a better name. I guess they are just left over from a debugging session to make them easily readable. No need to worry about unhappyness from my side.